tests/aaaaaaaaaaaaaaaaaaaaaa/lero.js

```diff
@@ -0,0 +1,28 @@
+a {
```

Comment:
The name of this file

- why .js and not .css?
- why “lero”?
- why “aaaaaaaaaaaaaa”?

Reply:
Just saw that @vjeux commented on this

Reply:
@lydell sorry about this, I like to create a file to minimize the repro case. "aaaaaaaa" is meant to be the first folder in the `tests/` folder. the ".js" is because the default is me working on JS and this is what's in `git stash`.

I just forgot removing this before creating the commit 😅

Reply:
Btw, we have `/test.*` ignored in `.gitignore` so if you want to have a dummy file you can save `/test.js` and do `./bin/prettier.js test.js`.

Reply:
@azz wow that's gonna make my life so much easier. Thanks!